config/set/code-quality.php

```diff
@@ -201,5 +202,6 @@
         TernaryEmptyArrayArrayDimFetchToCoalesceRector::class,
         OptionalParametersAfterRequiredRector::class,
         SimplifyEmptyCheckOnEmptyArrayRector::class,
+        ConvertStaticPrivateConstantToSelfRector::class,
```

Comment:
please don't add to config set list yet to avoid regression, we can add after some verify. You can ignore in phpstan config if it show the notice.

Reply:
I removed that, but it was reported by Rector and not PhpStan. That's why I needed to modify `utils/Command/MissingInSetCommand.php`. Let me know if this change is not what you wanted